v2/pkg/reporting/trackers/jira/jira.go

```diff
@@ -91,6 +94,52 @@ func (i *Integration) CreateIssue(event *output.ResultEvent) error {
 	return nil
 }
 
+// CreateIssue creates an issue in the tracker or updates the existing one
+func (i *Integration) CreateIssue(event *output.ResultEvent) error {
+	if i.options.UpdateExisting {
+		issue_id, err := i.FindExistingIssue(event)
+		if err != nil {
+			return err
+		} else if issue_id != "" {
+			_, _, err = i.jira.Issue.AddComment(issue_id, &jira.Comment{
+				Body: jiraFormatDescription(event),
+			})
+			return err
+		}
+	}
+	return i.CreateNewIssue(event)
+}
+
+// FindExistingIssue checks if the issue already exists and returns its ID
+func (i *Integration) FindExistingIssue(event *output.ResultEvent) (string, error) {
+	template := format.GetMatchedTemplate(event)
+	jql := fmt.Sprintf("summary ~ \"%s\" AND summary ~ \"%s\" AND status = \"Open\"", template, event.Host)
+
+	search_options := &jira.SearchOptions{
```

Comment:
please follow the linter guidelines on variable naming (camelCase instead of snake_case)